geomstats/geometry/riemannian_metric.py

```diff
@@ -518,13 +526,46 @@ def normal_basis(self, basis, base_point=None):
 
         return gs.einsum("i, ikl->ikl", 1.0 / gs.sqrt(norms), basis)
 
+    def covariant_riemann_tensor(self, base_point):
+        r"""Compute purely covariant version of Riemannian tensor at base_point.
+
+        In the literature the covariant riemannian tensor is noted R_{ijkl}.
+
+        Convention used in the literature (tensor index notation, ref. Wikipedia) is:
+        R_{ijkl} = <R(x_k, x_l)x_j, x_i> where x_i is the i-th basis vector of the
+        tangent space at base point.
+
+        [cov_riemann_tensor]_{ijkl} = [metric_matrix]_{im} [riemann_tensor]_{jkl}^m
```

Comment:
Use `:math:`.

Reply:
this one is a mix of math equation and variables' names: I've left it like this